GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/GVFSLockTests.cs

```diff
@@ -63,7 +63,9 @@ public void GitCheckoutFailsOutsideLock()
             }
         }
 
+        // Mac and Windows only because Linux uses a separate repo mount device

```

Comment:
Minor: Rather than adding this comment everywhere that tests are tagged as `RepositoryMountsSameFileSystem`, I think you could move it to the code that enables or disables RepositoryMountsSameFileSystem tests based on which platform we're on.

Reply:
I can definitely do that -- and maybe add a comment in `Categories.cs` as well.

Reply:
Done in d28ed04!